e2e/testdata/fn-eval/non-krm-resource/.expected/config.yaml

```diff
@@ -19,4 +19,5 @@ args:
 exitCode: 1
 stdErr: |
   [FAIL] "gcr.io/kpt-fn/set-namespace:v0.1.3" in 0s
-  error: input resource list must contain only KRM resources: nonkrm.yaml: resource must have `apiVersion`
+    Stderr:
+      "input resource list must contain only KRM resources: nonkrm.yaml: resource must have `apiVersion`"
```

Comment:
The output on this changes as well with this PR.

Currently:
```
$ kpt fn eval e2e/testdata/fn-eval/non-krm-resource/ --image=gcr.io/kpt-fn/set-namespace:v0.1.3 -- namespace=staging 
[RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1.3"
[FAIL] "gcr.io/kpt-fn/set-namespace:v0.1.3" in 0s
error: "input resource list must contain only KRM resources: nonkrm.yaml: resource must have `apiVersion`"
```

With this PR:
```
$ kpt fn eval e2e/testdata/fn-eval/non-krm-resource/ --image=gcr.io/kpt-fn/set-namespace:v0.1.3 -- namespace=staging 
error: [RUNNING] "gcr.io/kpt-fn/set-namespace:v0.1.3"
[FAIL] "gcr.io/kpt-fn/set-namespace:v0.1.3" in 0s
  Stderr:
    "input resource list must contain only KRM resources: nonkrm.yaml: resource must have `apiVersion`"
```